Automattic/wp-calypso
# Remove Quick Start upsell from /me/purchases

#### Changes proposed in this Pull Request

* This PR stops the Quick Start banner in `/me/purchases` from upselling users on Quick Start sessions by removing that mode from the concierge banner component
* The PR keeps all the reminder nudges in place for the time being

#### Testing instructions

* Load this branch locally or via the live branch
* For a user who doesn't have an active Business or ECommerce plan, and hasn't purchased a Quick Start session before, navigate to `/me/purchases`
* Verify that you don't see the Quick Start upsell banner

#### Screenshot

For clarity, this is the banner that is removed by this PR:
<img width="1159" alt="Screenshot 2021-11-04 at 11 14 41" src="https://user-images.githubusercontent.com/3376401/140288680-a4abe54f-3fe2-498b-a40b-0d907d0f04e7.png">
